reference/tax_settings.v3.yml

```diff
@@ -143,41 +143,22 @@ components:
-          description: This setting applies only if a merchant enters tax-inclusive prices. When enabled, the store subtracts the itemʼs store tax rate before calculating tax using the shopperʼs tax zone. The tax-exclusive amount will be the same across all tax zones. When disabled, the tax-inclusive price remains the same across all tax zones; only the tax amount will vary based on the shopperʼs location. The tax-exclusive amount may vary among tax zones. These calculations are relevant for tax pricing and tax quotations that use basic tax.    
-    Tax_Settings_Req:
-      type: object
-      properties:
-        tax_entered_with_prices:
-          type: boolean
-          description: Whether prices entered on this store include a tax component or not.
-        price_display_settings:
-          type: object
-          description: Settings that describe how prices display at the global level.
-          properties:
-            show_inclusive_in_control_panel:
-              type: boolean
-              description: Whether to show prices as tax inclusive or tax exclusive in the BigCommerce control panel.
-            invoice_price_display_strategy:
-              type: string
-              description: 'Whether to show prices as tax inclusive or tax exclusive across all invoices, or use the shopperʼs tax zone for price display on invoices.'
-              enum:
-                - ZONE
-                - INCLUSIVE
-                - EXCLUSIVE
-        fallback_strategy:
-          type: string
-          description: 'Decribes the fallback behaviour that applies when a tax provider produces an error. A merchant may decide to use a flat 10% fallback tax rate, their basic tax settings, or to block the transaction until a successful result can be achieved.'
-          enum:
-            - FIXED
-            - BASIC
-            - DISABLE
         should_subtract_store_tax:
           default: true
           type: boolean
           description: This setting applies only if a merchant enters tax-inclusive prices. When enabled, the store subtracts the itemʼs store tax rate before calculating tax using the shopperʼs tax zone. The tax-exclusive amount will be the same across all tax zones. When disabled, the tax-inclusive price remains the same across all tax zones; only the tax amount will vary based on the shopperʼs location. The tax-exclusive amount may vary among tax zones. These calculations are relevant for tax pricing and tax quotations that use basic tax.
+        should_use_geolocation_to_determine_guest_shopper_tax_zone:
+          default: false
+          type: boolean
+          description: When enabled, the guest shopper's geolocation will be used to determine the relevant country-level tax zone for storefront tax estimation. The geolocation data will be used to determine the corresponding tax zone provided in basic tax - once the corresponding tax zone is found, the tax estimation will be applied to the store.
```

Comment:
```suggestion
          description: This setting sets whether a store uses a guest shopper's geolocation to estimate storefront tax. When enabled, the store identifies a country-level tax zone based on the geolocation. The store then applies the corresponding guest shopper's tax zone to tax estimations. When disabled, the store uses the `guest_shopper_tax_zone_id` field to determine the corresponding zone. 
```

Reply:
> to estimate storefront tax

Sounds a bit awkward to me in that there's no such thing as "storefront tax". Perhaps this speaks to confused language around `store` vs `storefront` too. Perhaps we can simplify slightly:

> This setting determines which tax zone a store uses to estimate tax for guest shoppers.

Reply:
> This setting applies to basic tax; tax zones must be provided in the basic tax set up.

This could be confused as not being relevant for automatic tax providers somehow. What do you think of dropping this sentence and leaving it as implied?

Alternatively a slight simplification to remove the `applies to basic tax`:

> Note: tax zones must be configured in the basic tax set up.

Replacing `provided` with `configured` at least seems like a minor improvement (given we have 'tax providers' as nearby terminology)🤔

Reply:
Good points, I just removed `storefront` and used `configured` instead.

I see. At first, I didn't realize that Tax Providers could also use geolocation. I'm thinking of:

> If the store uses basic tax, only the tax zones you configure are available.

The sentence emphasizes that it doesn't have to be basic tax. I also reworded it to "only the tax zones you configure are available" to make the meaning slightly more accurate

Reply:
Thanks @bc-andreadao.

It's awkward since all stores use basic tax, for tax pricing.

I understand that the main reason for including this additional note is that we aren't magically pulling numbers out of thin air, we want to guide the merchant that they need to define these (country-level) tax zones that we'll match the guest shopper to.

What about:

> Only the tax zones you configure can be matched to the guest shopper's geolocation.

🤔

Reply:
Thanks, sounds good 👍